stylesheets/_layout.settings.scss

```diff
@@ -371,7 +392,13 @@
 
 .tab__container.has-settings .tab__inner {
     @include respond-min($screen-desktop) {
+        align-items: flex-start;
         display: table;
+        display: -webkit-box;
+        display: -moz-box;
+        display: -ms-flexbox;
+        display: -webkit-flex;
+        display: flex;
```

Comment:
More vendor here that we should be able to remove ☝️

Reply:
Providing the auto-prefixer takes care of this?

Reply:
It does, they've been removed